File: kotlin/src/main/kotlin/advertisements/advertisement/application/publishAdvertisement/PublishAdvertisementCommand.kt
package advertisements.advertisement.application.publishAdvertisement

data class PublishAdvertisementCommand(
    val securityUserId: String,
    val securityUserRole: String,
    val id: String,
    val description: String,
    val email: String,
    val password: String,
    val memberNumber: String,
    val civicCenterId: String
)

File: typescript/src/advertisements/advertisement/application/command/publish-advertisement/PublishAdvertisementCommand.ts
export class PublishAdvertisementCommand {
  constructor(
      public readonly securityUserId: string,
      public readonly securityUserRole: string,
      public readonly id: string,
      public readonly description: string,
      public readonly email: string,
      public readonly password: string,
      public readonly memberNumber: string,
      public readonly civicCenterId: string
  ) {}
}
